www/rustup.css

```diff
@@ -31,49 +31,69 @@
     src: local('Work Sans Medium'), url("fonts/WorkSans-Medium.ttf") format('truetype');
 }
 
+@font-face {
+    font-family: 'Alfa Slab One';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Alfa Slab One'),
+        url("fonts/AlfaSlabOne-Regular.woff2") format('woff2'),
+        url("fonts/AlfaSlabOne-Regular.woff") format('woff');
+}
+
 body {
-    margin-top: 2em;
     background-color: white;
-    color: #515151;
-    font-family: "Fira Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
+    font-family: "Fira Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
     font-weight: 300;
     font-size: 25px;
+    margin: 0;
 }
 
 pre {
     font-family: "Fira Mono", ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
     font-weight: 400;
 }
 
-body#idx #pitch > a {
-    font-weight: 500;
-    line-height: 2em;
+header > div {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+header > div > h1 {
+    font-family: "Alfa Slab One", serif;
+    color: black;
+    font-size: 4rem;
+    margin-bottom: 0;
+    margin-top: 1rem;
+    line-height: 1;
+    font-weight: 300;
+    letter-spacing: 1px;
+}
+
+header > div > h2 {
+    color: black;
+    font-size: 2rem;
+    font-weight: 300;
 }
 
 a {
-    color: #428bca;
-    text-decoration: none;
+    color: #0b7261;
```

Comment:
Why change the link color/decoration?

Reply:
Okay, so I think it makes sense to have at least one commit per bullet point, more strongly linking the commit message to the changes at a more granular level.

Reply:
Done